clang/test/Modules/module-file-info-mtime.m

```diff
@@ -0,0 +1,24 @@
+// Hardcode the mtime for a input file & check that is emitted in pcm during a implicit module invocation.
+
+// RUN: rm -fr %t
+// RUN: split-file %s %t
+// RUN: touch -m -a -t 202508011501 %t/BuildDir/A/A.h
```

Comment:
Are you sure this is cross-platform enough for LIT? If post-commits bots complain, I think I'd be enough to check that we print non-0 mtime below.